src/templates/mt5/economic_calendar.html.tt

```diff
@@ -6,7 +6,7 @@
 </div>
 
 <div class="container">
-    <p>[% l('You can monitor macroeconomic indicators straight from the MetaTrader 5 platform using release time labels displayed on <a href="[_1]" target="_blank">financial symbol charts</a>.', request.url_for('/download/MetaTrader5_Help.pdf')) %]</p>
+    <p>[% l('You can monitor macroeconomic indicators straight from the MetaTrader 5 platform using release time labels displayed on <a href="https://www.metatrader5.com/en/terminal/help/fundamental?utm_source=www.mql5.com&utm_campaign=en.main.page.mql5#chart" target="_blank">financial symbol charts</a>.') %]</p>
```

Comment:
The link can be simply: `https://www.metatrader5.com/en/terminal/help/fundamental#chart`